### PR TITLE
Adds http_method_text

### PR DESCRIPTION
**Pull request by Aglezabad**
Based on the advice of @jorisvink in #115.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ kore
 *.module
 *.DSYM
 cert
+.idea
```
Comment (jorisvink) [inline .gitignore]: This can probably be removed from the PR.

**Comment by jorisvink** [comment]
The style in this PR does not match the rest of the style in Kore. Please fix this first :-)

File: .gitignore (hunk 1 above)
Comment (Aglezabad) [inline .gitignore]: That line is because I'm using CLion, and I don't want to add garbage to parent repo.

Comment (Aglezabad) [inline .gitignore]: Ok, I think that it's better to not using CLion for Kore contributions. I will fix those style issues.

**Comment by Aglezabad** [comment]
I think that style is ok now. My error was using auto indentation with spaces instead of tabs.

**Comment by jorisvink** [comment]
Neat.

There's still 2 things bugging me style-wise. You have an empty line with an indented tab which shouldn't be there (the tab, newlines are fine). The variable definition for r does not follow suit. Look at http_status_text().

**Comment by Aglezabad** [comment]
Done. I think that style is perfect now.
I think that a style guide will be necessary for future contributions. If I have some time, I will write it.
Thank you for helping me.

**Comment by jorisvink** [comment]
No worries.

Can you just squash this into a single commit? No need for 6 different commits with each unrelated commit messages for a simple addition as simple like this.

.joris

**Comment by jorisvink** [comment]
I know I'm being difficult :-) just bear with me I'm trying to keep the log history clean.

**Comment by Aglezabad** [comment]
Well, I didn't made a squash of multiple commits, but I've tried it. I hope that this is ok.

**Comment by jorisvink** [comment]
Neat thanks.

**Pull request merged**